styles.css

```diff
@@ -0,0 +1,58 @@
+* {
+	margin: 0;
+	padding: 0;
+}
+.options {
+  position: fixed;
+  left: 15px;
+  top: 15px;
+  width: 330px;
+  z-index: 1;
+}
+.main-container {
+	background: url("refrigerator.png") no-repeat center center;
+	height: 1545px;
+	width: 700px;
+	margin: 0 auto;
+  position: relative;
+  z-index: 0;
+}
+.magnet {
+  margin: 5px;
```

Comment:
Having the cursor change to the text icon is weird especially since you disable selection. You might think about using cursor:move or cursor:pointer here.